project_scrum/models/project.py

```diff
@@ -0,0 +1,65 @@
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    sprint_ids = fields.One2many(
+        comodel_name="project.sprint",
+        inverse_name="project_id",
+        string="Sprints",
+    )
+    sprint_count = fields.Integer(compute="_compute_sprint_count")
+    backlog_count = fields.Integer(compute="_compute_backlog_count")
+
+    def _compute_backlog_count(self):
+        for project in self:
+            project.backlog_count = len(
+                project.task_ids.filtered(
+                    lambda task: not task.sprint_id and task.state != "1_done"
+                )
+            )
+
+    def _compute_sprint_count(self):
+        unassigned_sprint_count = self.env["project.sprint"].search_count(
+            [("project_id", "=", False)]
+        )
+        for project in self:
+            project.sprint_count = len(project.sprint_ids) + unassigned_sprint_count
+
+    def action_sprints(self):
+        self.ensure_one()
+        return {
+            "name": self.env._("Sprints"),
+            "type": "ir.actions.act_window",
+            "res_model": "project.sprint",
+            "view_mode": "tree,form,timeline",
```

Comment:
```suggestion
            "view_mode": "list,form,timeline",
```

![image](https://github.com/user-attachments/assets/c8237236-e57a-4016-a954-19efac6c1aca)